README.md

```diff
@@ -1,51 +1,88 @@
+* **Node.js**\
+  _JavaScript Runtime, um JavaScript auch ausserhalb des Browsers einsetzen zu können_
+* **React**\
+  _JavaScript Bibliothek für Benutzerschnittstellen_
+* **Typescript**\
+  _JavaScript + statische Typisierung für sicheren Code_
+* **Docker**\
+  _Virtualisierung_
 
 ## Voraussetzungen
+
 Die diversen Softwarepakete welche eingesetzt werden, werden im Verlauf des Tutorials installiert. Wer das im voraus machen möchte, folgende Software wird eingesetzt:
-* GIT (https://git-scm.com/downloads)
+
+* Git (https://git-scm.com/downloads)
 * Node.js (https://nodejs.org)
 * Docker (https://www.docker.com/get-started)
 
-Alle eingesetzte Software ist Plattformunabhängig und funktioniert auf Windows/Linux/OSX. Ich persönlich empfehle, Linux zu verwenden. Insbesondere Docker läuft unter Linux wesentlich besser als auf Windows oder OSX.
+Alle eingesetzte Software sind plattformunabhängig und funktionieren unter Windows, Linux und macOS. 
+
+**Ich persönlich empfehle, Linux zu verwenden. Insbesondere Docker läuft unter Linux wesentlich besser als unter Windows oder macOS.**
+
+## Git installieren
+
+### Windows
 
-## GIT Installieren
-### Windows:
 Auf https://git-scm.com/downloads den Installer herunterladen und alles installieren (die Default Einstellungen sind in Ordnung).
 
-Nachher am besten immer die GIT Bash vom Startmenu für die weiteren Befehle verwenden. Die GIT Bash ist Linux-Kompatibel. Mir sind Befehle in Zusammenhang mit Node.js bekannt, welche in der Vergangenheit Probleme gemacht wenn die PowerShell oder CMD verwendet wird.
-Ausserdem sind die Befehle, welche später im Zusammenhang mit Docker verwendet werden ebenfalls Bash-Shell kompatibel. Von daher macht es sowieso sinn, sich damit vertraut zu machen.
+Nebst Git wird auch die **Git Bash** installiert und ich empfehle diese Konsole für die weiteren Schritte zu verwenden. Die Git Bash ist Linux-Kompatibel. Mir sind Befehle in Zusammenhang mit Node.js bekannt, welche in der Vergangenheit Probleme gemacht wenn die PowerShell oder CMD verwendet wird.
+
+Ausserdem sind die Befehle, welche später im Zusammenhang mit Docker verwendet werden ebenfalls Bash-Shell kompatibel. Von daher macht es sowieso Sinn, sich damit vertraut zu machen.
 
 ### Linux (Debian/Ubuntu)
+
 ```console
 $ sudo apt install git
 ```
 
+## Node.js installieren
+
+### Windows
 
-## Node.js Installieren
-### Windows:
 Auf https://nodejs.org den Installer herunterladen und alles installieren (die Default Einstellungen sind in Ordnung).
 
-Nachher ausloggen oder den Computer neustarten. Erst nachher wird die Umgebungsvariable PATH neu geladen, welche im Installationsvorgang (bei GIT und Node.js) angepasst wird.
+Nachher ausloggen oder den Computer neustarten. Erst nachher wird die Umgebungsvariable `PATH` neu geladen, welche im Installationsvorgang (bei Git und Node.js) angepasst wird.
 
 ### Linux (Debian/Ubuntu)
+
 Der Anleitung auf https://nodejs.org/en/download/package-manager/ befolgen.
```

Comment:
Wieso auf die Anleitung verweisen, du schreibst ja in der Überschrift schon, dass es für "Debian und Ubuntu" ist?